rollup.config.mjs

```diff
@@ -7,24 +7,14 @@ import postcss from 'rollup-plugin-postcss';
  */
 export default {
   input: 'src/index.ts',
-  output: [
-    {
-      dir: 'lib',
-      format: 'es',
-      preserveModules: true,
-      preserveModulesRoot: 'src',
-      sourcemap: true,
-    },
-    {
-      dir: 'lib',
-      format: 'cjs',
-      preserveModules: true,
-      preserveModulesRoot: 'src',
-      sourcemap: true,
-      entryFileNames: '[name].cjs',
-      interop: 'auto',
-    },
-  ],
+  output: {
+    dir: 'lib',
+    format: 'cjs',
+    preserveModules: true,
+    preserveModulesRoot: 'src',
+    sourcemap: true,
+    interop: 'auto',
+  },
```

Comment:
For posterity: there's an interesting thing here (including in the current EDS) where rollup is tree shaking out `index.js` files, but leaving the types for them...

So for example, the lib/Button directory ends up looking like this:

```
lib/Button
  Button.js
  Button.d.ts
  index.d.ts <- There's an index.d.ts, but no corresponding index.js!
```

But, crucially, the import/export statements in the output are such that everything still works.

Basically, when doing `import {Button} from 'eds'`
- Node or bundlers load Button.js directly
- TypeScript thinks you're loading Button through index.js (which doesn't exist)
- But everything still works

Just an oddity I noticed and wanted to note.

We could stop that behavior by adding `treeshake: false` to the rollup config. Doing that results in everything lining up, but prints some warnings and I'm not sure if there are any ramifications or not.

Reply:
@ahuth we can create a ticket to disable tree-shaking and to fix the warnings, if that's the right direction to go for now? thoughts?

Reply:
Good idea. I'll create it.